app/code/Magento/Captcha/Test/Unit/Observer/CheckUserLoginBackendObserverTest.php

```diff
@@ -0,0 +1,178 @@
+
+    /**
+     * @var ManagerInterface|MockObject
+     */
+    private $messageManagerMock;
+
+    /**
+     * @var CaptchaStringResolver|MockObject
+     */
+    private $captchaStringResolverMock;
+
+    /**
+     * @var RequestInterface|MockObject
+     */
+    private $requestMock;
+
+    /**
+     * @var Data|MockObject
+     */
+    private $helperMock;
+
+    /**
+     * Set Up
+     *
+     * @return void
+     */
+    protected function setUp()
+    {
+        $this->helperMock = $this->createMock(Data::class);
+        $this->messageManagerMock = $this->createMock(ManagerInterface::class);
+        $this->captchaStringResolverMock = $this->createMock(CaptchaStringResolver::class);
+        $this->requestMock = $this->createMock(RequestInterface::class);
+
+        $this->observer = new CheckUserLoginBackendObserver(
+            $this->helperMock,
+            $this->captchaStringResolverMock,
+            $this->requestMock
+        );
+    }
+
+    /**
+     * Test check user login in backend with correct captcha
+     *
+     * @dataProvider captchaCorrectnessCheckDataProvider
+     * @param bool $isRequired
+     * @param bool $isCorrect
+     * @param int $invokedTimes
+     * @return void
+     * @throws AuthenticationException
+     */
+    public function testCheckOnBackendLoginWithCorrectCaptcha(bool $isRequired, bool $isCorrect, int $invokedTimes): void
+    {
+        $formId = 'backend_login';
+        $login = 'admin';
+        $captchaValue = 'captcha-value';
+
+        /** @var Observer|MockObject $observerMock */
+        $observerMock = $this->createPartialMock(Observer::class, ['getEvent']);
+        $eventMock = $this->createPartialMock(Event::class, ['getUsername']);
+        $captcha = $this->createMock(DefaultModel::class);
+
+        $eventMock->expects($this->any())
+            ->method('getUsername')
+            ->willReturn('admin');
+        $observerMock->expects($this->any())
+            ->method('getEvent')
+            ->willReturn($eventMock);
+        $captcha->expects($this->once())->method('isRequired')
+            ->with($login)
+            ->willReturn($isRequired);
+        $captcha->expects($this->exactly($invokedTimes))
+            ->method('isCorrect')
+            ->with($captchaValue)
+            ->willReturn($isCorrect);
+        $this->helperMock->expects($this->once())
+            ->method('getCaptcha')
+            ->with($formId)
+            ->willReturn($captcha);
+
+        $this->captchaStringResolverMock->expects($this->exactly($invokedTimes))
+            ->method('resolve')
+            ->with($this->requestMock, $formId)
+            ->willReturn($captchaValue);
+
+        $this->messageManagerMock->expects($this->exactly(0))
+            ->method('addError')
+            ->with(__('Incorrect CAPTCHA'));
+
+        $this->observer->execute($observerMock);
+    }
+
+    /**
+     * @return array
+     */
+    public function captchaCorrectnessCheckDataProvider(): array
```

Comment:
Change to `correctCaptchaDataProvider`, no need to pass `isCorrect` as it is invariant.